examples/azure/terraform-azure-actiongroup-example/main.tf

```diff
@@ -0,0 +1,60 @@
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY AN ACTION GROUP
+# This is an example of how to deploy an Azure Action Group to be used for Azure Alerts
+# ---------------------------------------------------------------------------------------------------------------------
+
+# ------------------------------------------------------------------------------
```

Comment:
NIT: This is not the same length as the other block comments - we should update this so it is consistent.

Reply:
Updated.